src/main/web_src/src/components/ui/form/kategori/kategori.less

```diff
@@ -1,12 +1,15 @@
 .form-kategori {
-	h4 {
-		display: flex;
-		align-items: center;
-		margin: 15px 0 10px;
-	}
 
-	&-flex {
-		display: flex;
-		flex-wrap: wrap;
-	}
+  width: 100%;
```

Comment:
Fikset problemet, men litt omfattende endring kanskje? 
Tenker at det bør være greit, en kategori skal vel alltid ta opp 100% av bredden i en flexboks uansett eller? 😅

Reply:
Gjorde susen den der da! 😄  Og såvidt jeg kan se ødelegger det ikke for kategori andre steder - for som du sier så skal den uansett fylle bredden 😊